scripts/simtest/cargo-simtest

```diff
@@ -96,6 +96,10 @@ fi
 # Must supply a new temp dir - the test is deterministic and can't choose one randomly itself.
 export TMPDIR=$(mktemp -d)
 
+# Set the example move package for the simtest static initializer
+# https://github.com/MystenLabs/sui/blob/7bc276d534c6c758ac2cfefe96431c2b1318ca01/crates/sui-proc-macros/src/lib.rs#L52
+export SIMTEST_STATIC_INIT_MOVE="../../examples/move/basics"
```

Comment:
I would recommend using `git rev-parse --show-toplevel` to get the repo root, and then making this an absolute path

Reply:
Sounds good!